### PR TITLE
Created Ansible role for PostgreSQL metrics-exporter

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

**Review by fridex — COMMENTED** [review]
LGTM. :+1: 


I wonder if we should not host container images of components we do not develop (such as PostgreSQL, this metrics-exporter, ...) in our quay org.

File: tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+  - name: "create route for PostgreSQL metrics-exporter"
```
Comment (harshad16) [inline tasks/main.yml:14]: My idea is to add the following task in here and use the password from openshift secret:
```
    - name: Get PostgreSQL encrypted secrets
      shell: "oc get secrets postgresql -n {{ namespace }} -o=jsonpath='{.data.database-password}'"
      register: postgresql_pwd_encrypt

    - name: Decrypt PostgreSQL encrypted secrets
      shell: "echo {{ postgresql_pwd_encrypt.stdout.strip() }} |base64 -d"
      register: postgresql_pwd
      when: postgresql_pwd_encrypt is defined

    - set_fact:
        postgresql_password: "{{ postgresql_pwd.stdout }}"```

Comment (harshad16) [reply tasks/main.yml:14]: @pacospace  @fridex what do you think about this? 
Reference to: https://github.com/thoth-station/thoth-ops/pull/119#discussion_r344772303

Comment (pacospace) [reply tasks/main.yml:14]: Updated :)

Comment (fridex) [reply tasks/main.yml:14]: @harshad16 User and database will match the defaults as done in the postgresql role. What about namespace used during provisioning (stage/test environment, ...)?

Comment (harshad16) [reply tasks/main.yml:14]: The namespace would be passed by **thoth-ops** provision script which uses this role, so we manage stage/test from there.
About the  other user and database, yes there are default one. In hindsights, it looks like may be having these values explicitly assigned in thoth-ops and using them looks like a good option.

**Review by harshad16 — APPROVED** [review]

**Pull request merged**